src/components/Pagination/Pagination.stories.js

```diff
@@ -22,7 +22,7 @@ const PaginationTemplate = (args) => ({
     },
   },
   template: `
-    <div style="padding: 20px">
+    <div style="padding: 20px; margin-top: 250px">
```

Comment:
Why is this necessary?

Reply:
This is for the the preview on storybook, not change the style for the real component, it's for the better position of the pagination on the preview.